Formula/conduit.rb

```diff
@@ -0,0 +1,19 @@
+class Conduit < Formula
+  desc "Streams data between data stores. Kafka Connect replacement. No JVM required"
+  homepage "https://conduit.io/"
+  url "https://github.com/ConduitIO/conduit/archive/refs/tags/v0.5.0.tar.gz"
+  sha256 "ba57506c17a99356c443d3c4459977c825dcce1b039965ba7699140e11d95afc"
+  license "Apache-2.0"
+  head "https://github.com/ConduitIO/conduit.git", branch: "main"
+
+  depends_on "go"
+
+  def install
+    system "make", "build-server"
+    bin.install "conduit"
+  end
+
+  test do
+    shell_output("#{bin}/conduit -version").match(/0.5.0/)
```

Comment:
```suggestion
    assert_match version.to_s, shell_output("#{bin}/conduit -version")
```
But we need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
@SMillerDev  we have `make test` that runs all the unit tests in the code, would that work?

Reply:
No, it should be something the user would run too. You can have 100% unit test coverage and still have a broken install.

Reply:
ok I'll run the binary `conduit` and make sure both the gRPC and the HTTP servers started, by matching shell output